static/js/admin/feature_form.js

```diff
@@ -22,36 +22,21 @@ form.addEventListener('change', function(e) {
         intentStageChanged(e.target);
       }
       break;
-    case 'input':
-      if (e.target.name === 'shipped_milestone') {
-        fillOperaFields(e.target);
-      }
-      break;
     default:
       break;
   }
 });
 
-const operaDesktop = document.querySelector('#id_shipped_opera_milestone');
-const operaAndroid = document.querySelector(
-  '#id_shipped_opera_android_milestone');
+
+const specLink = document.querySelector('#id_spec_link');
 
 /**
- * Populates Opera version inputs with Chrome 32 -> Opera 19 version mapping.
- * @param {HTMLInputElement} chromeField Chrome version input.
+ * Toggles the spec link input.
+ * @param {HTMLInputElement} stdStage Input element.
  */
-function fillOperaFields(chromeField) {
-  const chromeVersion = chromeField.valueAsNumber;
-  if (chromeVersion < 28) {
-    return;
-  }
-  const operaVersion = chromeVersion - 13; // e.g. Chrome 32 ~ Opera 19
-  if (!operaDesktop.classList.contains('interacted')) {
-    operaDesktop.value = operaVersion;
-  }
-  if (!operaAndroid.classList.contains('interacted')) {
-    operaAndroid.value = operaVersion;
-  }
+function toggleSpecLink(stdStage) {
```

Comment:
There's linting errors with this. It's never called?

Reply:
Actually, we removed this in https://github.com/GoogleChrome/chromium-dashboard/commit/c4fc8bdd97cb48c6d5bb0c35d9507185ab6e57c5

Reply:
Done.